templates/aws/registry_nodes/manifest.yaml

```diff
@@ -0,0 +1,63 @@
+name: aws-registry-nodes
+description: |
+  Creates ec2 instances in AWS with a Route53 record to their public ip for registries.
+variables:
+  aws_access_key:
+    sensitive: true
+    type: string
+    optional: false
+    description: "aws API access key"
+  aws_secret_key:
+    sensitive: true
+    type: string
+    optional: false
+    description: "aws API secret key"
+  aws_region:
+    type: string
+    optional: false
+    description: "The region the ec2 instance would be created"
+  aws_ami:
+    type: string
+    optional: false
+    description: "The image of the ec2 instance"
+  aws_route53_zone:
+    type: string
+    optional: false
+    description: "The route 53 hosted zone"
+  aws_ssh_user:
+    type: string
+    optional: false
+    description: "The user need to initally ssh into an instance i.e ubuntu"
+  aws_security_group:
+    type: string
+    optional: false
+    description: "The security group needed for an ec2 instance"
+  aws_vpc:
+    type: string
+    optional: false
+    description: "The vpc of where the target group is located"
+  aws_subnet:
+    type: string
+    optional: false
+    description: "The subnet where the ec2 instance and the load balancer is created"
+  instance_type:
+    type: string
+    optional: false
+    description: "What size ec2 instance to use for the nodes."
+  registry_fqdn:
+    type: string
+    readOnly: true
+    description: "Public DNS A record that directs to the registry node."
+  registry_ip:
+    type: string
+    readOnly: true
+    description: "Public IP address of the registry node"
+  registry_count:
```

Comment:
Will be cases with more than one registry node?

Reply:
I think this is a remnant from when I begun following the `agent_nodes`' design.
But then stepped back on that after I realized that's not the same case for registries.

I'll remove it.

Reply:
Removed unused variable
No re-test needed.